emulated-integration-tests/src/chains/pop_network/genesis.rs

```diff
@@ -0,0 +1,55 @@
+// Substrate
+use sp_core::storage::Storage;
+
+// Cumulus
+use emulated_integration_tests_common::{
+	accounts, build_genesis_storage, collators, SAFE_XCM_VERSION,
+};
+use parachains_common::Balance;
+
+pub const PARA_ID: u32 = 909;
+pub const ED: Balance = rococo_runtime_constants::currency::EXISTENTIAL_DEPOSIT / 10;
```

Comment:
This should be imported from the runtime: https://github.com/r0gue-io/pop-node/blob/main/runtime/src/lib.rs#L226